src/renderer/shared/components/Forms/Button/Button.js

```diff
@@ -6,7 +6,7 @@ import styles from './Button.scss';
 
 export default class Button extends React.Component {
   render() {
-    return (
+    return ( // eslint-disable-next-line react/button-has-type
```

Comment:
could you add in the type here as well, just like the other buttons? :P

Reply:
The type is in here implicitly.  It's part of the `props`, which we splat here with the spread operator.  If you check out the `defaultProps` further down in this file, you can see it's set to "button".

Reply:
eslint doesn't recognize it though, which is why I added the `eslint-disable-next-line` comment here.